### PR TITLE
Feature: Add python scripts to generate initial state

### PR DESCRIPTION
**Pull request by lwojtow**

### CODE REVIEW

File: tools/account_info.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import json
+from constants import GENESIS_ACCOUNT, TEST_ACCOUNT, JSON_RPC_URL
+from time import sleep
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests.account_info import AccountInfo
+from xrpl.models.response import ResponseStatus
+
+# Creates client talking to a node in our testnet.
+client = JsonRpcClient(JSON_RPC_URL)
+
```
Comment (niklaslong) [inline tools/account_info.py:11]:
```suggestion
```

File: tools/account_info.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        return False
+    print("response.status: ", response.status)
+
+    # If request successful, prints information about the account.
+    if response.status == ResponseStatus.SUCCESS:
+        print(json.dumps(response.result, indent=4, sort_keys=True))
+        return True
+    else:
+        return False
+
```
Comment (niklaslong) [inline tools/account_info.py:35]:
```suggestion
```

**Review by niklaslong — APPROVED** [review]

File: tools/account_info.py (hunk 1 above)
Comment (lwojtow) [reply tools/account_info.py:11]: This generates warning in IDE. (Python has this strange idea that empty line matters).

File: tools/account_info.py (hunk 2 above)
Comment (lwojtow) [reply tools/account_info.py:35]: As above - warning in IDE.

**Pull request merged**